lib/screens/transfer/send/send_enter_data/interactor/send_enter_data_bloc.dart

```diff
@@ -62,7 +65,9 @@ class SendEnterDataPageBloc extends Bloc<SendEnterDataPageEvent, SendEnterDataPa
         ),
       );
 
-      yield SendTransactionMapper().mapResultToState(state, result);
+      final bool shouldShowInAppReview = await inAppReview.isAvailable();
+
```

Comment:
Why not?
`final bool shouldShowInAppReview = await InAppReview.instance.isAvailable();`

Reply:
I think that is a singleton

Reply:
we dont need it as a var, that was a mistake.  fixed